### ISSUE TITLE
Prevent hotswapping elytra and armor from the action bar while tagged

### ISSUE THREAD

**Issue by Nanpure** [SirBlobman/CombatLogX#820]
### Server Implementation

Spigot

### Server Version

other

### CombatLogX Version

Java Version: 21.0.1
Java Vendor: Red Hat, Inc.
Java URL: N/A
Server Version: git-Paper-462 (MC: 1.20.4)
Bukkit Version: 1.20.4-R0.1-SNAPSHOT
Minecraft Version: 1.20.4
NMS Version: 1_20_R3
Dependency Information:
 - AngelChest (not installed)
 - ASkyBlock (not installed)
 - BentoBox (not installed)
 - Citizens (not installed)
 - CMI (not installed)
 - CrackShot (not installed)
 - CrashClaim (not installed)
 - Essentials v2.21.0-dev+24-0af4436
 - FabledSkyBlock (not installed)
 - FeatherBoard (not installed)
 - FlagWar (not installed)
 - GriefDefender (not installed)
 - GriefPrevention v17.0.0-23-g000d884-dirty
 - HuskHomes (not installed)
 - HuskSync (not installed)
 - HuskTowns (not installed)
 - iDisguise (not installed)
 - IridiumSkyblock (not installed)
 - Kingdoms (not installed)
 - Konquest (not installed)
 - Lands (not installed)
 - LibsDisguises (not installed)
 - MarriageMaster (not installed)
 - MCPets (not installed)
 - MythicMobs (not installed)
 - PlaceholderAPI v2.11.5
 - PlayerParticles (not installed)
 - PreciousStones (not installed)
 - PremiumVanish (not installed)
 - ProtectionStones (not installed)
 - ProtocolLib v5.2.0-SNAPSHOT-679
 - RedProtect (not installed)
 - Residence (not installed)
 - Sentinel (not installed)
 - SuperiorSkyblock2 (not installed)
 - SuperVanish v6.2.18
 - Towny (not installed)
 - UltimateClaims (not installed)
 - uSkyBlock (not installed)
 - VanishNoPacket (not installed)
 - WorldGuard v7.0.9+5934e49
 - ViaVersion (not installed)
 - BlueSlimeCore v2.9.5.Beta-407
CombatLogX by SirBlobman
Local Version: 11.4.0.2.Beta-1227
Remote Version: 11.4.0.1.1193
Enabled Expansions (21):
 - Action Bar v17.3
 - Boss Bar v17.1
 - Cheat Prevention v17.6
 - EssentialsX Compatibility v17.2
 - GriefPrevention Compatibility v17.1
 - LuckPerms Compatibility v17.1
 - PlaceholderAPI Compatibility v17.2
 - SuperVanish Compatibility v17.1
 - WorldGuard Compatibility v17.2
 - Damage Effects v17.0
 - Damage Tagger v17.1
 - Death Effects v17.0
 - End Crystal Helper v17.1
 - Force Field v17.4
 - Glowing v17.0
 - Logger v17.3
 - Loot Protection v17.1
 - Mob Tagger v17.1
 - Newbie Helper v17.2
 - Rewards v17.4
 - Scoreboard v17.1

### Plugin

None

### Explanation

Players are asking for hot swapping between armor from the action bar to be restricted to prevent their prey from easily swapping their chestplate for an elytra and flying away. We do not want this to be part of inventory restriction, since we want players to have access to their inventory, but not rapid hot swapping from the action bar. This means that if they want to put their elytra on, it will take precious seconds as they open their inventory to drag the item manually. However, an option to toggle equipping of any armor while tagged would be a useful option, although separate.

In summary, two separate features (although overlapping):

Disallow swapping armor directly from the action bar
Disallow swapping armor *at all*, which is inclusive of the action bar and the inventory screen
